pages/linux/repo-add.md

```diff
@@ -0,0 +1,15 @@
+# repo-add
+
+> Package database maintenance utility which enables installation via pacman.
```

Comment:
Does this mean that:

1. It enables me to install a given package file with `pacman`
2. pacman can't work without it

Reply:
Number 1. I'll cleanup that part.

Reply:
https://github.com/tldr-pages/tldr/pull/6622/commits/3aadcab4afac46d4fed555d76dfcb65fa95db507 I hope the following commit helps.